### ISSUE TITLE
add argument to turn off plotting in evaluate_alteration function

### ISSUE THREAD

**Issue by ryanpeek** [ceff-tech/ffc_api_client#31]
just a note for batch processing, we probably need to turn off the plotting option. Default can be that plotting occurs, but should add argument for `plotting=FALSE` to allow just the data to get pulled in...and then maybe option to run plotting function separately? I can work on this but wanted to remember.

**Comment by nickrsan** [comment]
Yeah, I think that makes sense - I think these two functions should remain as the core shortcut functions that go through the whole workflow, so being able to disable certain aspects while going through the rest makes sense so that the alternative isn't to basically recreate the function without that one thing. Noted!

**Comment by nickrsan** [comment]
I'm going to add the high priority label to this so that we can have it ready in time to do the next set of batching.

**Comment by nickrsan** [comment]
Added to evaluate_gage_alteration in d09b8f00b16c8a2aded7f4e0089522110950f3ef, but still want to add it to evaluate_alteration to have a consistent API between the two.
